specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/JavaComponents.json

```diff
@@ -490,6 +490,22 @@
             "propertyName"
           ]
         },
+        "scale": {
+          "description": "Java component scaling configurations.",
+          "type": "object",
+          "properties": {
+            "minReplicas": {
```

Comment:
shall we describe the default value, as what ContainerApp does https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2024-08-02-preview/specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/CommonDefinitions.json#L591

Reply:
Done